dev.local/launch.json

```diff
@@ -0,0 +1,22 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```

Comment:
Remove comments from json file. https://stackoverflow.com/questions/244777/can-comments-be-used-in-json

Reply:
done